core/src/main/java/org/elasticsearch/search/aggregations/AggregationBuilder.java

```diff
@@ -69,6 +70,16 @@ public String getName() {
     /** Add a sub aggregation to this builder. */
     public abstract AggregationBuilder subAggregation(PipelineAggregationBuilder aggregation);
 
+    /** Return the configured set of subaggregations **/
+    public List<AggregationBuilder> getSubAggregations() {
```

Comment:
Ah sorry, should have explained.  I need/needed this for something I'm working on in XPack.  So it'll definitely be used, but my dependent code is still in work-in-progress state :)